Formula/tnftpd.rb

```diff
@@ -0,0 +1,63 @@
+class Tnftpd < Formula
+  desc "NetBSD's FTP server (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftpd/lukemftpd-51.tar.gz"
+  sha256 "969b8a35fabcc82759da2433973b9606b7b62f73527e76ac8f18d0a19f473c2a"
+
+  depends_on :xcode => :build
+
+  def install
+    system "tar zxvf tnftpd-20100324.tar.gz"
+    Dir.chdir "tnftpd-20100324"
+
+    system "./configure"
+    system "make"
+
+    File.rename "src/tnftpd", "src/ftpd"
```

Comment:
These lines are all redundant. You can simply do things like:
```ruby
bin.install "src/tnftpd" => "ftpd
```
etc

Reply:
@DomT4 Ok, thanks. Couldn't figure out how exactly to get that sort of thing to work.